api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1556,6 +1556,22 @@ public void setId(Integer id) {
 		}
 		return Collections.unmodifiableList(conceptMembers);
 	}
+
+	/**
+	 * @param includeRetired true/false whether to also include the retired concepts
+	 * @return List&lt;Concept&gt; the Concepts that are members of this Concept's set
+	 * @should return the same as getSetMembers() if includeRetired is true
```

Comment:
In the past, we use these should annotations to generate the corresponding unit tests. If you are not using them for such, we can get rid of them.

Reply:
Can you we also add a @since annotation?